### DISCUSSION TITLE
[Custom Layout] Forza Horizon 4 and 5 / MotorSports

### DISCUSSION THREAD

**Discussion by Tak-attack** [redphx/better-xcloud#242] [Custom touch controls]
**Game title**
Forza Horizon 4
https://www.xbox.com/ja-JP/play/games/forza-horizon-4-%E6%A8%99%E6%BA%96%E7%89%88/9PNJXVCVWD4K

Forza Horizon 5
https://www.xbox.com/ja-JP/play/games/forza-horizon-5-%E6%A8%99%E6%BA%96%E7%89%88/9NKX70BBCDRN

Forza MotorSports (2023)
https://www.xbox.com/ja-JP/play/games/forza-motorsport-standard-edition/9PLKVSWR299F

Forza Horizon 4 / 5
A: Hand Brake
B: Shift up
X: Activate
Y: Rewind
RB: Switch Camera
LB: Clutch
RT: Accelerate
LT: Brake / Reverse
LS: Steer
LS Thumb: none
RS: Camera
RS Thumb: Horn
D-pad: Quick action
Menu: Menu
View: Open Map

Forza MotorSports (2023)
A: Hand Brake
B: Shift up
X: Shift down
Y: Rewind
RB: Switch Camera
LB: Clutch
RT: Accelerate
LT: Brake / Reverse
LS: Steer
LS Thumb: none
RS: Camera
RS Thumb: TTS input
D-pad: Quick action
Menu: Menu
View: Pit Entry Assist / Reset to Track

The operation is slightly different between Horizon and MotorSports, but the layout arrangement should be the same with only the icons changed.

**Reference**
I present an example of Dirt5.
It should be noted that when the Accelerate button is pressed, if you slide your finger on the edge of the button, the handbrake is pulled together.
This is useful because it allows two simultaneous inputs with one button.
Alternatively, you can use RT (Accelerate) as a toggle input, but it may be difficult to control the fine acceleration. (It might be nice to have two different gas pedals, a main hold button and a sub toggle button.)

Dirt5
https://www.xbox.com/ja-JP/play/games/dirt-5/9PJGM0T0827V
![1706638260181](https://github.com/redphx/better-xcloud/assets/149154024/a7c37355-cd7b-4248-9611-18187b4389f2)

Also, since the game does not use the D-pad frequently, I think Tunic's D-pad layout switching button would be helpful. It seems to me that it would be easier to operate if a larger D-pad is displayed on the other side only while the switch button is pressed.

Tunic
https://www.xbox.com/ja-JP/play/games/tunic/9NLRT31Z4RWM
【Example Movie】
https://github.com/redphx/better-xcloud/assets/149154024/676a936b-bd75-4ad5-adb8-310d601741ca

These layouts assume Default LAYOUT 1.
However, since the Forza series allows free customization of controls, it may be useful to leave some unused buttons such as LSThumb for placement.

**Comment by redphx** [comment]
Added for FH 4 & 5 (only tested in 5)

![Forza Horizon 5](https://github.com/redphx/better-xcloud/assets/96280/4b8e51be-0f80-468f-bbf0-92663471050a)

**Reply by Tak-attack** [reply]
Thanks! I have tried it on Horizon 4 and it seems to work well.

**Comment by redphx** [comment]
Added Forza Motorsport, same layout as Horizon

![FM](https://github.com/redphx/better-xcloud/assets/96280/accd85bd-d7d7-4a2d-85ab-94a28aaa2a6f)
